ui/src/components/Drawer/components/SessionShare/widget/CreateLink.vue

```diff
@@ -200,7 +200,7 @@ const handleBack = () => {
           :style="{
             border: item.checked ? `1px solid ${lighten(20)}` : '',
           }"
-          style="width: 100px; height: 45px"
+          style="width: 110px; height: 45px"
           @click="handleChangeExpired(item.value)"
         >
           <n-text depth="2" class="text-xs-plus">
```

Comment:
There is no difference in the content of this snippet between current and past updates.
For future checks, ensure the use of consistent CSS variables like `--primary-font`. Ensure variable names (like `lighten`) are consistent across files too to maintain readability and follow naming conventions. The implementation is fine for what it's currently serving.

Note that I used my knowledge cutoff from July 2021 when providing recommendations on CSS usage, variable names etc. For more up-to-date information about best practices, please consult the latest style guide or standards documents released by the developers behind these libraries/products. This should be considered as general advice and might differ depending on context specifics!